manifests/plugin/container.pp

```diff
@@ -2,21 +2,17 @@
 # @param location_prefix
 #   In the Apache configuration a location with this prefix is exposed. The
 #   version (currently v2) will be appended.
-# @param registry_version_path
-#   The path beneath the location prefix to forward. This is also appended to
-#   the content base url.
 class pulpcore::plugin::container (
   String $location_prefix = '/pulpcore_registry',
-  String $registry_version_path = '/v2/',
```

Comment:
My understanding is that flatpak needs a different path, so instead of having two locations to proxy, it was decided to proxy the whole `/pulpcore_registry`.

Reply:
Correct. https://github.com/theforeman/puppet-pulpcore/pull/363#pullrequestreview-2476687309 tells me it needs to retrieve `/pulpcore_registry/index/static` though the spec says there's also `/pulpcore_registry/index/dynamic`. Prior to https://github.com/theforeman/puppet-pulpcore/pull/363#discussion_r1876286574 there were both, but that looks redundant to me.